### ISSUE TITLE
Contribution Badges

### ISSUE THREAD

**Issue by robmoffat** [finos/technical-oversight-committee#165]
An idea from @smendis-scottlogic: 

- FINOS should issue badges to both firms and individuals contributing to FINOS projects.  
- Badges could be distributed via cred.ly, as with other LF badges
- Individuals would be able to share these on LinkedIn etc. which would be a useful way of publicising when firms and individuals are building things for FINOS.
- This may have the effect of making it easier to find volunteers for projects, both from firms and individuals, since there is a clear link to recognition.
- TOC may be a good body to oversee and administer this.

Suggested schedule:

| Badge     |  Firm   | Individual |
|-----------|---------|---------|
| Bronze 🥉   | 2 days / pcm.  |2 hrs / pcm |
| Silver  🥈 | 4 days / pcm.  |4 hrs / pcm |
| Gold 🥇  | 8 days / pcm.  |8 hrs / pcm |

This would need to be based largely on honour, but the TOC could conduct spot-checks to make sure contributions are happening.

**Comment by eddie-knight** [comment]
@eminty69 Conceptually, this seems like something a TOC Liaison would request on behalf of their respective projects to encourage and recognize ongoing contributions

**Comment by eminty69** [comment]
@eddie-knight I think the contribution badges are a really good idea but is there no way to allocate them automatically?  Having the TOC liaison do a review of who/how many contributions for a project makes sense but a manual process where the TOC liaison has to request the badge seems like it might be a bit uneven.

**Comment by eddie-knight** [comment]
Perhaps maintainers can request the badges, and the Liaison can approve? Then part of the quarterly review can be an eyeball check to make sure the badges are up to date. WDYT?

**Comment by psmulovics** [comment]
I am all for 'developer flair', it has been a big success at many different places I worked / volunteered / etc at. The easier (but still trackable and comfortably trustable) we can make this process, the better benefit for all.

**Comment by robmoffat** [comment]
cred.ly has an API so I don't think an automation is off the table once we get this up and running and working how we want it

**Comment by eminty69** [comment]
I think it has to be automated but we need a working process first.  I would say that having the quarterly review makes sense but I'd probably go with the project maintainers allocating the badges. This is going to need to be honour based at some level so the maintainers should have the best view on who is doing what.  The review can then look at consistency of approach with the liaison and will also feed into how active the projects are.

**Comment by mindthegab** [comment]
This an amazing idea. I would think to start lightweight first, by simply having a "maintainer" badge (those who actually have write access to the repo) and a "contributor" badge (anyone who has a pull request merged). Starting to measure PCM opens a lot can of worms IMHO or even having to involve the TOC seems very time consuming.

Given that, assuming projects follow the standard [software project governance ](https://github.com/finos/software-project-blueprint/blob/main/CONTRIBUTING.md#maintainer-voting)), maintainers are the one nominating other maintainers, and the FINOS team changes their permissions on the repo, @TheJuanAndOnly99 @opoupeney and team could actually start by manually issuing this badge as part of their new maintainer onboarding SOPs. And if it becomes time consuming we couid automate.

Just my 0.02$ :)

**Comment by caldeirav** [comment]
I agree in direction with @mindthegab thought process - starting with a maintainer badge (based on community role) and contributor badge based on commits is the easiest way to get started. We can find a mechanism for maintainers to issue contributor badge ideally, and think about additional special badges for key contributors down the line once the process is working.

**Comment by eddie-knight** [comment]
@TheJuanAndOnly99 — Where are we currently tracking official project maintainers?

**Comment by eddie-knight** [comment]
The TOC has voted on today's call to create the following recognition:

- Official project maintainers may request a recognition for contributors who make significant impact in the form of time, effort, or tangible outcome.
- The request must me made through the project's regular TOC Project Liaison channels
- The recognition will be issued by FINOS, executed by FINOS staff, in the form of a Credly badge or equivalent
- Pending routine approval, the TOC issues a standing request that these recognitions are summarized in community calls
